gen/templates/profile_parcels/resource_test.go

```diff
@@ -185,9 +185,13 @@ func testAccSdwan{{camelCase .Name}}ProfileParcelConfig_all() string {
 				{{- if len .TestTags}}
 	if {{range $i, $e := .TestTags}}{{if $i}} || {{end}}os.Getenv("{{$e}}") != ""{{end}} {
 		config += `			{{.TfName}} = {{if .TestValue}}{{.TestValue}}{{else}}{{if eq .Type "String"}}"{{.Example}}"{{else if isStringListSet .}}["{{.Example}}"]{{else if isInt64ListSet .}}[{{.Example}}]{{else}}{{.Example}}{{end}}{{end}}` + "\n"
+		{{if .SecondaryTestValue}}config += `		}, {` + "\n"
```

Comment:
Can you try changing this to `{{- if ...` to see if that fixes the empty lines being inserted in the otherwise unchanged rendered files?

Reply:
This is good to know!